File: src/main/java/com/dongtronic/diabot/data/nutrition/FullNutrient.java

package com.dongtronic.diabot.data.nutrition;

import java.util.HashMap;
import java.util.Map;
import com.fasterxml.jackson.annotation.JsonAnyGetter;
import com.fasterxml.jackson.annotation.JsonAnySetter;
import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonPropertyOrder;

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonPropertyOrder({
    "attr_id",
    "value"
})
public class FullNutrient {

    @JsonProperty("attr_id")
    private Integer attrId;
    @JsonProperty("value")
    private Integer value;
    @JsonIgnore
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    @JsonProperty("attr_id")
    public Integer getAttrId() {
        return attrId;
    }

    @JsonProperty("attr_id")
    public void setAttrId(Integer attrId) {
        this.attrId = attrId;
    }

    @JsonProperty("value")
    public Integer getValue() {
        return value;
    }

    @JsonProperty("value")
    public void setValue(Integer value) {
        this.value = value;
    }

    @JsonAnyGetter
    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    @JsonAnySetter
    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }

}

File: src/main/java/com/dongtronic/diabot/data/nutrition/FullNutrient.kt
package com.dongtronic.diabot.data.nutrition

import com.fasterxml.jackson.annotation.*
import java.util.*

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonPropertyOrder("attr_id", "value")
class FullNutrient {
    @get:JsonProperty("attr_id")
    @set:JsonProperty("attr_id")
    @JsonProperty("attr_id")
    var attrId: Int? = null
    @get:JsonProperty("value")
    @set:JsonProperty("value")
    @JsonProperty("value")
    var value: Int? = null
    @JsonIgnore
    private val additionalProperties: MutableMap<String, Any> = HashMap()

    @JsonAnyGetter
    fun getAdditionalProperties(): Map<String, Any> {
        return additionalProperties
    }

    @JsonAnySetter
    fun setAdditionalProperty(name: String, value: Any) {
        additionalProperties[name] = value
    }
}